src/main/java/jewellery/inventory/dto/request/ProductPriceDiscountRequestDto.java

```diff
@@ -0,0 +1,19 @@
+package jewellery.inventory.dto.request;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@RequiredArgsConstructor
+@AllArgsConstructor
+public class ProductPriceDiscountRequestDto {
+  @NotNull private UUID productId;
+   private double salePrice;
+   private double discount;
```

Comment:
What happens if the user decides NOT to pass salePrice and discount in the request? Because the variables are double, it's going to assign them the default value - 0.0. I imagine we are going to set the price and discount of the product to 0. This shouldn't be the case. In this case we should not change the price and discount of the product. Let's make them `Double` and make checks if the values are not passed (they are null), then we wouldn't change them in the product.